apps/amiapp_flutter/ios/Podfile

```diff
@@ -41,7 +41,7 @@ target 'Runner' do
   
   # Uncomment only 1 of the lines below to install a version of the iOS SDK
   pod 'CustomerIO/MessagingPushFCM', '~> 2' # install production build
-  # install_non_production_ios_sdk_local_path(local_path: '~/code/customerio-ios/', is_app_extension: false, push_service: "fcm")
+  install_non_production_ios_sdk_local_path(local_path: '~/code/customerio-ios/', is_app_extension: false, push_service: "fcm")
```

Comment:
This change can be reverted before merging into main. Needed during development to test native iOS SDK.